docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    axs[0].plot(x[idx_motifA:idx_motifA+m], T[idx_motifA:idx_motifA+m])
</pre><p>Then, please replace <code>idx_motifA</code> with <code>idx</code>. </p><p> </p><p>Please revise the other lines accordingly as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>